### DISCUSSION TITLE
Save/log movies not matched in list?

### DISCUSSION THREAD

**Discussion by qwadr8** [mza921/Plex-Auto-Collections#229] [Q&A]
Hello! 
What an awesome application! 

Thank you so much! 

I have one not that important question however. If I match i.e. BFI Top 100 movies-list with my library and get 60 movies match. Is there any way to get a result list of the 40 movies not matched? I don't have the movies but I get obsessed with the idea to get them for collection-completion :D I could of course go thru this manually and see what is not matching, if there's some solution that I have missed, that would be nice :) Thanks again!

**Comment by meisnate12** [comment]
development here has stalled check out [Plex Meta Manager](https://github.com/meisnate12/Plex-Meta-Manager) its a complete rewrite of Plex Auto Collections. It has the ability to display movies not added to the collections as well as many many bug fixes and other feature improvements
